src/Jab/ContainerGenerator.cs

```diff
@@ -595,9 +600,12 @@ public override void Initialize(AnalysisContext context)
 
     private static string ReadAttributesFile()
     {
+        const string placeholder = "#if JAB_ATTRIBUTES_PACKAGE";
```

Comment:
Do you mind adding the attribute directly in the Attributes.cs file?

Reply:
What about the version number?

Reply:
[It looks like](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/CodeDom/Compiler/GeneratedCodeAttribute.cs#L15) the version parameter is nullable. I'd be fine with passing `null`.

Reply:
Done